File: cielo24_Java/cielo24/src/cielo24/json/JobInfo.java
package cielo24.json;

import cielo24.utils.Guid;
import java.util.ArrayList;
import com.google.gson.annotations.SerializedName;

public class JobInfo extends JsonBase {
	@SerializedName("JobId")
	public Guid jobId;
	@SerializedName("JobName")
	public String jobName;
	@SerializedName("Language")
	public String language;
	@SerializedName("Tasks")
	public ArrayList<Task> tasks;
}

File: cielo24_.NET/Cielo24/JSON/Job/JobInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Newtonsoft.Json;

namespace Cielo24.JSON.Job
{
    public class JobInfo : JsonBase
    {
        [JsonProperty("JobId")]
        public Guid JobId { get; set; }
        [JsonProperty("JobName")]
        public string JobName { get; set; }
        [JsonProperty("Language")]
        public string Language { get; set; }
        [JsonProperty("Tasks")]
        public List<Task> Tasks { get; set; }
    }
}
